template/src/components/Button/Button.tsx

```diff
@@ -3,6 +3,7 @@ import clsx from 'clsx';
 
 type ButtonProps = {
   buttonContent: ReactNode | string;
+  isSubmit?: boolean;
```

Comment:
Suggestion passing down `type` will be more flexible

Reply:
TypeScript throws errors with any use of type beyond static string and ternary expression. Ugh.

Reply:
@n3ps I resolved this by using the type and disabling the Eslint rule